packages/emergency/src/lib/components/id/emergency-id/emergency-id.component.ts

```
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { EmergencyUi } from '@tevet-troc-client/models';
+import { TextComponent, TextDirective } from '@tevet-troc-client/text';
+import { BadgeComponent } from '@tevet-troc-client/badge';
+import { JumbotronComponent } from '@tevet-troc-client/jumbotron';
+import { InfoLineComponent } from '@tevet-troc-client/info-line';
+
+@Component({
+  selector: 'lib-emergency-id',
```

ESLint

[error] The selector should start with one of these prefixes: "app" (https://angular.dev/style-guide#style-02-07)

(@angular-eslint/component-selector)